nvtabular/ops.py

```diff
@@ -773,7 +777,10 @@ def op_logic(self, gdf: cudf.DataFrame, target_columns: list, stats_context=None
         cont_names = target_columns
         if not cont_names or not stats_context["stds"]:
             return
-        gdf = self.apply_mean_std(gdf, stats_context, cont_names)
+        if self.ntype == "mean_std":
+            gdf = self.apply_mean_std(gdf, stats_context, cont_names)
+        elif self.ntype == "min_max":
+            gdf = self.apply_min_max(gdf, stats_context, cont_names)
```

Comment:
I think we should create a new operator rather than have both minmax scaling and mean/std normalization in the same class. @jperez999 what do you think?

Reply:
Yes, each of those stats should be its own operator. The idea is to make stat operators that represent a single statistic. This allows users to clearly understand what stats are available in a workflow. It will also ensure minimal workflow size when all repeated operators are removed.